docs/contributing/releases.md

```diff
@@ -16,23 +16,38 @@ and it is being used in nix environments.
+- [ ] generate the list of pull requests finished since the last release using the [haskell script](https://github.com/haskell/haskell-language-server/blob/master/GenChangelogs.hs) in the project root.
   Nix users should run command `gen-hls-changelogs` (a wrapper of the script) in nix-shell instead.
-* [ ] add that list to the actual [Changelog](https://github.com/haskell/haskell-language-server/blob/master/ChangeLog.md) with a description of the release.
-* [ ] bump up versions of changed packages. All are optional but [haskell-language-server itself](https://github.com/haskell/haskell-language-server/blob/master/haskell-language-server.cabal).
-* [ ] create the tag and make an initial prerelease to trigger the ci workflow (see details below)
-* [ ] contact ghcup team (#haskell-ghcup irc channel or via its [repo](https://gitlab.haskell.org/haskell/ghcup-hs/-/issues)) to try to sync our release and its inclusion in ghcup
-* [ ] check uploaded binaries (see windows note below) and the release description (usually the changelog entry) and uncheck the prerelease box
-* [ ] make public the release in the usual social channels: irc, twitter, reddit, discord, discourse, mailing lists, etc (not required but useful to spread the word :slightly_smiling_face:)
-
+- [ ] add that list to the actual [Changelog](https://github.com/haskell/haskell-language-server/blob/master/ChangeLog.md) with a description of the release.
+- [ ] bump up versions of changed packages. All are optional but [haskell-language-server itself](https://github.com/haskell/haskell-language-server/blob/master/haskell-language-server.cabal).
+- [ ] create the tag and make an initial prerelease to trigger the ci workflow (see details below)
+- [ ] contact ghcup team (#haskell-ghcup irc channel or via its [repo](https://gitlab.haskell.org/haskell/ghcup-hs/-/issues)) to try to sync our release and its inclusion in ghcup
+- [ ] check uploaded binaries (see windows note below) and the release description (usually the changelog entry) and uncheck the prerelease box
+- [ ] make public the release in the usual social channels (not required but useful to spread the word :slightly_smiling_face:):
+  - [ ] irc
+  - [ ] matrix
+  - [ ] twitter
+  - [ ] discord
+  - [ ] discourse
+  - [ ] reddit
 ### hackage release
 
-* [ ] bump up package versions following the [pvp specification](https://pvp.haskell.org/) if they are not already updated
-* [ ] create ${version}-hackage branch to trigger the hackage github workflow which will upload all changed packages to hackage as candidates
-* [ ] check manually candidates in hackage
-* [ ] publish them definitely
+- [ ] bump up package versions following the [pvp specification](https://pvp.haskell.org/) if they are not already updated. You could use [policeman](https://github.com/kowainik/policeman) to help with this step.
```

Comment:
how is this different to the previous verison-bumping step on line 29? should there just be one version-bumping step done before any releasing?

Reply:
The difference is in the `All are optional but haskell-language-server itself` part. For hackage bump up versions following pvp is mandatory. For the github release you only need to bump up hls itself.
At the beginning we did not have the github and hackage release in sync and we only bumped up plugin and package versions in a systematic way for the hackage one. In last releases i bumped up versions before start the release engine.